internal/models/users.model.go

```diff
@@ -35,3 +37,14 @@ type LoginRequest struct {
 	Email    string `json:"email"`
 	Password string `json:"password"`
 }
+
+type EmailVerificationData struct {
+	Email     string     `json:"email"`
+	Code      string     `json:"code"`
+	ExpiredAt *time.Time `json:"expire_at"`
```

Comment:
Remove `*` from `*time.Time`. Last time we added this because that field in database can be `null` which in this case, I believe, cannot.
The following is what I understand when using pointer field.
struct type | db | struct value
-|-|-
time.Time | null | default value of time.Time
*time.Time | null | nil

Reply:
omg
Roger that